handbook/engineering/distribution/index.md

```diff
@@ -45,33 +45,33 @@ Distribution team members may also be involved in other areas of Sourcegraph not
     - Pure-docker shell scripts & upgrade docs
     - Cloud-specific setup docs (AWS/Google Cloud)
     - Deployment setup & upgrade docs
-    - **Primary owners**: @stephen, @geoffrey
+    - **Primary owners**: @uwe, @geoffrey
     - **Related code**: [deploy-sourcegraph-docker repository](https://github.com/sourcegraph/deploy-sourcegraph-docker), [docker-compose installation docs](https://sourcegraph.com/github.com/sourcegraph/sourcegraph/-/tree/doc/admin/install/docker-compose), [docker-compose upgrade docs](https://sourcegraph.com/github.com/sourcegraph/sourcegraph/-/blob/doc/admin/updates/docker_compose.md) [pure-docker upgrade docs](https://sourcegraph.com/github.com/sourcegraph/sourcegraph/-/blob/doc/admin/updates/pure_docker.md).
   - **Single-container installation & upgrade experience**
     - Primarily in maintenance mode
     - Pushing admins to upgrade to Docker Compose
     - Communicating the limitations of single-container deployments
-    - **Primary owners**: @stephen
+    - **Primary owners**: @uwe, @geoffrey
     - **Related code**: [cmd/server in main repo](https://sourcegraph.com/search?q=repo:%5Egithub%5C.com/sourcegraph/sourcegraph%24+file:cmd/server/&patternType=regexp)
   - **Scalability**
     - Documenting when to upgrade from one deploy type to another
     - Resource estimation for new deployments
     - Scaling advice for existing deployments
-    - **Primary owners**: @stephen
+    - **Primary owners**: TBD
```

Comment:
All of these ownership areas are described above as:

> It aims to be 100% comprehensive, but the owners are merely aspirational goal posts, not mandates. At the end of the day, whoever has most context will own the area.

I just replaced all instances with people who I think have the most context aside from me, but this one is a clear deficiency area. Uwe has second most substantial context, but not a ton.

As an aside, is this "primary owners" communication here actually useful? I'm not sure

Reply:
I think it could be detrimental, I would just refer to the distribution team.

Reply:
```suggestion
    - **Primary owners**: whole team, @stephen for advice
```